deploy/complete/helm-chart/mushop/charts/carts/values.yaml

```diff
@@ -21,7 +21,7 @@ secrets:
 env:
   zipkin: zipkin.jaeger.svc.cluster.local
   javaOpts: -Xms64m -Xmx128m -XX:+UseG1GC -Dlogging.level.mushop.carts=TRACE  -Djava.security.egd=file:/dev/urandom -Dspring.zipkin.enabled=false
-  tnsAdmin: "/app/config"
+  tnsAdmin: "/usr/lib/oracle/19.3/client64/lib/network/admin/"
```

Comment:
Why this is listed on values? `/usr/lib/oracle/19.3/client64/lib/network/admin/` is the standard for the Db Driver. There's any use case where the deployer would need to change that?